### DISCUSSION TITLE
Built in LinearGradient

### DISCUSSION THREAD

**Discussion by aaronrine** [react-native-community/discussions-and-proposals#463] [Ideas]
Gradients are fairly common styling tool in many websites and apps, but a built in way to create one doesn't exist afaik for react native. Is there a reason RN doesn't come with either a View style prop for linear gradients or a built in component for it? I'm aware of react-native-linear-gradient, it's just odd that this isn't part of the standard install and I'd like to understand more about why that is.

**Comment by cortinico** [comment]
This is aligned to the lean core effort:
- https://github.com/facebook/react-native/issues/23313

We intentionally moved away a lot of components/modules to community maintained one, to avoid bloating the codebase and shipping code that some users were not using at all. So if you need to use a Gradient, you can add `react-native-linear-gradient` and use it from there 👍
